chandra_aca/maude_decom.py

```diff
@@ -567,18 +573,27 @@ def get_raw_aca_packets(start, stop, maude_result=None, **maude_kwargs):
     )  # ensure input is proper date
     stop_pad = 1.5 / 86400  # padding at the end in case of trailing partial ACA packets
 
+    # get the frames and unpack front matter
+    if maude_result is None:
+        frames = maude.get_frames(
+            start=date_start, stop=date_stop + stop_pad, **maude_kwargs
+        )["data"]
+    else:
+        frames = maude_result["data"]
+    flags = frames["f"]
+    frames = frames["frames"]
+
     # also getting major and minor frames to figure out which is the first ACA packet in a group
-    vcdu_counter = maude.get_msids(
-        ["CVCMNCTR", "CVCMJCTR", "CVCDUCTR"],
-        start=date_start,
-        stop=date_stop + stop_pad,
-        **maude_kwargs,
+    w = np.array([1, 2**8, 2**16])[::-1]
```

Comment:
This part also probably deserves a comment?  The code is a little opaque here too.